modules/mobile/docs/openapi.yaml

```diff
@@ -1683,6 +1166,8 @@ paths:
               schema:
                 $ref: ./schemas/LighthouseErrors.yml
           description: Failure in processing entity
+        '408':
```

Comment:
Most of the additions you've added have been for "universal" responses like 403 and 502. Any reason you're adding 408 and 422 here? Did you get these by cross-checking against the lighthouse docs?